service/api/service.go

```diff
@@ -683,6 +683,14 @@ func (s *serviceImpl) ApiV1WorkflowDumpPost(ctx context.Context, request iwfidl.
 	}, nil
 }
 
+func (s *serviceImpl) ApiInfoHealth(ctx context.Context) *iwfidl.HealthInfo {
+	return &iwfidl.HealthInfo{
+		Condition: iwfidl.PtrString("OK"),
+		Hostname:  iwfidl.PtrString("will check"),
```

Comment:
It would be nice if you can use `os.Hostname()` to get the hostname

Reply:
Please check